### ISSUE TITLE
Schedule Confirmation page does not show details - Alpha 14.1

### ISSUE THREAD

**Issue by tbunney** [SparkDevNetwork/Rock#5284]
Rock Version: Rock McKinley 14.1 (1.14.1.1)
Client Culture Setting: en-US

![accept-schedulingconfirmation](https://user-images.githubusercontent.com/37702418/212968516-c4015b43-a726-4977-9b20-03f1585dd38d.PNG)

![decline-schedulingconfirmation](https://user-images.githubusercontent.com/37702418/212968567-fc796781-9eac-4ad2-a526-2b1e94912f38.PNG)

### Steps to Reproduce

1. Go to Group Scheduling
2. Send a Group Scheduling email.
3. Accept or decline the email.

**Expected behavior:**

A confirmation page with all Group Scheduling details, with person's name, location, and schedule they are serving.

**Actual behavior:**

Location and Schedule are missing

### Versions

* **Rock Version:** Rock McKinley 14.1 (1.14.1.1)
* **Client Culture Setting:**  en-US

**Comment by nairdo** [comment]
@tbunney We're having some trouble reproducing this -- can you include the URL portion of the screenshot so we can see if there are missing parameters in there (having to do with the AttendanceId/AttendanceIds).

**Comment by tbunney** [comment]
Confirm Lava: <h2 class='margin-t-none'>{{ Person.NickName }}, you’re confirmed to serve.</h2><p>Thanks for letting us know.  You’re confirmed for:</p><p><b>{{ OccurrenceDate | Date:'dddd, MMMM d, yyyy' }}</b><br>{{ Group.Name }}<br>{{ ScheduledItem.Location.Name }} {{ScheduledItem.Schedule.Name }} <i class='text-success fa fa-check-circle'></i><br></p>
<p class='margin-b-lg'>Thanks again!<br>
{{ Scheduler.FullName }}</p>

Decline Lava: <h2 class='margin-t-none'>{{ Person.NickName }}, can’t make it?</h2><p>Thanks for letting us know.  We’ll try to schedule another person for:</p>
<p><b>{{ OccurrenceDate | Date:'dddd, MMMM d, yyyy' }}</b><br>
{{ Group.Name }}<br>
{{ ScheduledItem.Location.Name }} {{ ScheduledItem.Schedule.Name }}<br></p>

**Comment by nairdo** [comment]
The fix for this will be in the form of a data migration that will happen during the v14.1 install/update.   There are three block settings that will be automatically adjusted/replaced (as long they were not modified too heavily).

For example, as long as the **Decline Header Template** setting has:

```
{{ Group.Name }}<br>
{{ ScheduledItem.Location.Name }} {{ ScheduledItem.Schedule.Name }}<br></p>
```

Then it will be replaced with the newer, necessary Lava that loops over a list of `ScheduledItems`.  If for some reason the settings have been overly customized, it's still possible to just clear the block setting, save and then re-apply any customizations.

**Comment by nairdo** [comment]
Verified fixed on the v14.1 beta which was just applied to the Rock demo site.
![image](https://user-images.githubusercontent.com/383807/214689713-a8d79450-ac2e-4758-9b73-427212b6aae4.png)
